src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/SharedBlobListenerFactory.cs

```diff
@@ -13,8 +13,9 @@ internal class SharedBlobListenerFactory : IFactory<SharedBlobListener>
         private readonly IStorageAccount _account;
         private readonly IWebJobsExceptionHandler _exceptionHandler;
         private readonly IContextSetter<IBlobWrittenWatcher> _blobWrittenWatcherSetter;
+        private readonly string _hostId;
 
-        public SharedBlobListenerFactory(IStorageAccount account,
+        public SharedBlobListenerFactory(string hostId, IStorageAccount account,
```

Comment:
my ignorance here, is the hostId web app specific? function specific? instance specific?

Is there only one instance which listens to blobs, or do we parallelize somehow?